.docker/bin/database.sh

```diff
@@ -0,0 +1,183 @@
+#!/bin/bash
+
+WP="wp --allow-root"
+DATA_PATH="/usr/src/data"
+DATA_FILENAME="wordcamp_dev.sql"
+
+# Entry point.
+#
+# $1 The database command to execute.
+#
+function main() {
+    local cmd="$1"
+
+    local orig_dir="$PWD"
+    cd /usr/src/public_html
+
+    case "$cmd" in
+        backup-current ) backup_current;;
+        clean )          clean;;
+        import )         import ${@:2};;
+        export )         export;;
+        restore )        restore ${@:2};;
+
+        # Clean up the current state of the database and then export it to the provision file.
+        clean-export )
+            clean
+            export
+        ;;
+
+        # Delete the current state of the database and then re-import the provision file.
+        reset )
+            _reset
+            import --skipbackup
+        ;;
+
+        help | * )
+            echo "Please enter a valid command."
+            echo
+            echo -e "  backup-current \n  clean \n  clean-export \n  import \n  export \n  reset \n  restore <file>"
+        ;;
+    esac
+
+    echo
+
+    cd $orig_dir
+}
+
+# Clean unnecessary data from the database.
+#
+# This currently includes post revisions and transients.
```

Comment:
`SHOULD` - It might be good to add a note here that this is only meant to be run in development environments where the data is already fake, and that it's *not* safe to run against production.

Actually, it might be even better to `echo` that to the console at the start of the script, rather than it being buried in a comment here.

Reply:
Good idea. abd93d7e

This adds an environment variable to the docker config that the database script looks for, and it bails with an explanatory message if it doesn't find it.